properties.schema

```diff
@@ -1,128 +1,846 @@
 {
```

Comment:
Obviously v5 is a major breaking change, but should the schema have been updated as much as it has been?

Reply:
The schema has been updated so the following is possible in the Authoring Tool:

Change colour variables
Page header background image
Page header minimum heights
Article background image
Article minimum heights
Block background image
Block minimum heights
Block divider boolean

The idea is to allow more customisation and theme editing through the Authoring Tool without having to upload a custom theme plugin.